local.cfg.EXAMPLE

```diff
@@ -145,26 +145,26 @@ db.schema = public
 
 # Canonical Handle URL prefix
 #
-# By default, DSpace is configured to use http://hdl.handle.net/
-# as the canonical URL prefix when generating dc.identifier.uri
-# during submission, and in the 'identifier' displayed in JSPUI
-# item record pages.
+# Items in DSpace receive a unique url, stored in dc.identifier.uri
+# after it is generated during the submission process.
 #
-# If you do not subscribe to CNRI's handle service, you can change this
-# to match the persistent URL service you use, or you can force DSpace
-# to use your site's URL, eg.
-#handle.canonical.prefix = ${dspace.url}/handle/
+# If you register with CNRI's handle service at http://www.handle.net/, 
+# these links can be generated as permalinks using the http://hdl.handle.net/
+# as the canonical prefix.
+#
+# After registering with handle.net, make sure your handle.canonical.prefix is set to:
+#handle.canonical.prefix = http://hdl.handle.net/
 #
 # Note that this will not alter dc.identifer.uri metadata for existing
 # items (only for subsequent submissions), but it will alter the URL 
 # in JSPUI's 'identifier' message on item record pages for existing items.
 #
 # If omitted, the canonical URL prefix will be http://hdl.handle.net/
-#handle.canonical.prefix = http://hdl.handle.net/
+handle.canonical.prefix = ${dspace.url}/handle/
 
 # CNRI Handle prefix
-# (Defaults to a dummy/fake prefix of 123456789)
-#handle.prefix = 123456789
+# After registering with handle.net, enter the prefix assigned to your organization here.
+handle.prefix = internal
```

Comment:
@bram-atmire : one thing I just realized.  The configs in `local.cfg.EXAMPLE` are commented out on purpose (because the defaults all reside in dspace.cfg, and *everything* is optional in `local.cfg`).  So, this PR likely needs also change default settings in dspace.cfg.  Otherwise, if a config isn't specified in your local.cfg, the existing defaults in dspace.cfg will be in affect.